### ISSUE TITLE
Arabic Processing data 

### ISSUE THREAD

**Issue by A7med7x7** [stanfordnlp/CoreNLP#1441]
<img width="319" alt="Screenshot 2024-04-16 at 10 59 36 PM" src="https://github.com/stanfordnlp/CoreNLP/assets/95593096/b11150cd-b1e6-44fb-ae11-865db549eb9c">

why the arabic processing data seems to be violence and records of criminal activity, like it's derived from a news API?

**Comment by AngledLuffa** [comment]
They usually are derived from news API.  I don't have any influence over
the data itself for Arabic

It'd be easier and faster for us to use more varied sources of data for
Stanza, but we can always redo either the Stanza or the CoreNLP models if
you have more data available

**Comment by A7med7x7** [comment]
@AngledLuffa 
Yes, you definitely don't have control but at least providing general data that is not biased towards one behaviour will help a lot, as all the samples I have seem to be this type. 
data is everywhere what's more important is to check it's validity and why it intuitively means, and I can help wtih that.
